src/util/__tests__/createServerRootMixin.test.js

```diff
@@ -414,6 +414,13 @@ Array [
         {
           helper: expect.any(Object),
           results: expect.any(Object),
+          scopedResults: expect.arrayContaining([
+            expect.objectContaining({
+              helper: expect.any(Object),
+              indexId: expect.any(String),
```

Comment:
Why don't we assert actual values?

Reply:
Since most of the values are coming from the test itself or from mocks then we are just checking the shape of the `RenderOptions` object, not its actual values.